Mathlib/CategoryTheory/Category/Factorisation.lean

```diff
@@ -112,11 +112,14 @@ instance : Unique (d ⟶ (Factorisation.terminal : Factorisation f)) where
 
 open Limits
 
-instance : IsInitial (Factorisation.initial : Factorisation f) := IsInitial.ofUnique _
+/-- The initial factorization is an initial object -/
+def IsInitial_initial : IsInitial (Factorisation.initial : Factorisation f) := IsInitial.ofUnique _
 
 instance : HasInitial (Factorisation f) := Limits.hasInitial_of_unique Factorisation.initial
 
-instance : IsTerminal (Factorisation.terminal : Factorisation f) := IsTerminal.ofUnique _
+/-- The terminal factorization is a terminal object -/
+def IsTerminal_terminal : IsTerminal (Factorisation.terminal : Factorisation f) :=
+IsTerminal.ofUnique _
```

Comment:
Hmm, rare non-american spelling of Factorisation in this file (including file name).

Reply:
I fixed my comments to conform. Apparently despite being from the UK I have become americanis(z)ed.

Reply:
Yes, there's some alignment to be done at some point, it's called "factorisation" in the category theory part of mathlib and "factorization" everywhere else.